aclk/aclk.c

```diff
@@ -461,9 +460,9 @@ static int aclk_block_till_recon_allowed() {
  */
 static int aclk_get_transport_idx(aclk_env_t *env) {
     for (size_t i = 0; i < env->transport_count; i++) {
-        // currently we support only MQTT 3
+        // currently we support only MQTT 5
         // therefore select first transport that matches
-        if (env->transports[i]->type == ACLK_TRP_MQTT_3_1_1) {
+        if (env->transports[i]->type == ACLK_TRP_MQTT_5) {
```

Comment:
This fails since only ACLK_TRP_MQTT_3_1_1 is received from the cloud

Reply:
Cloud change was deployed on production therefore I removed the `[WIP]` label